server/src/test/java/org/opensearch/index/translog/RemoteFsTranslogTests.java

```diff
@@ -1048,7 +1056,7 @@ public void testConcurrentWriteViewsAndSnapshot() throws Throwable {
             final int threadId = i;
             writers[i] = new Thread(new AbstractRunnable() {
                 @Override
-                public void doRun() throws BrokenBarrierException, InterruptedException, IOException {
+                public void doRun() throws Exception {
```

Comment:
Why do we want to not explicitly specify all the Exceptions thrown here?

Reply:
It is either ways same as the test would fail if an exception gets thrown. In this case, the IDE autoformat feature decided to replace n-many exceptions to just one Exception class.